code/datums/outfits/outfit_admin.dm

```diff
@@ -220,24 +220,40 @@
 
 /datum/outfit/admin/nt_undercover
 	name = "NT Undercover Operative"
+	var/access_level = "Emergency Response Team Leader"
 	// Disguised NT special forces, sent to quietly eliminate or keep tabs on people in high positions (e.g: captain)
+	//nerfed ghost version, doesn't get the op centcom combat implant, less access
 
 	uniform = /obj/item/clothing/under/color/black
 	back = /obj/item/storage/backpack
 	belt = /obj/item/storage/belt/utility/full/multitool
-	gloves = /obj/item/clothing/gloves/combat
+	gloves = /obj/item/clothing/gloves/color/yellow
 	shoes = /obj/item/clothing/shoes/chameleon/noslip
 	l_ear = /obj/item/radio/headset/centcom
 	id = /obj/item/card/id
 	pda = /obj/item/pda
 	backpack_contents = list(
 		/obj/item/storage/box/engineer = 1,
 		/obj/item/flashlight = 1,
-		/obj/item/pinpointer/crew = 1
+		/obj/item/pinpointer/crew = 1,
+		/obj/item/gun/projectile/automatic/proto = 1,
+		/obj/item/ammo_box/magazine/smgm9mm = 3,
```

Comment:
You might also want to give them a comms jammer, or something so that they don't instantly get dogpiled by sec when their subject screams on radio.

Reply:
Good idea. I thought about giving them an emp flashlight or something but that works, too.

Reply:
Radio jammer may not work, due to the fact they can still call out on comns and make a lot of radio noise jibberish, which will probably cause the AI to track them, and people nearby may notice the radio jammer and call sec. EMP flashlight or a muting chemical would probably work better.

Reply:
Could make a modified version of radio jammer that totally silences radio, instead of just garbling it.

Reply:
Also, they only have one spare mag. Once that's gone, they're stuffed.

Reply:
Uh, no they have 3 spare mags. If 3 spare mags and the one in the gun isn't enough to do the job it's less an undercover op and more a war.

Reply:
Oh, =3. Nevermind on that one.